readme.md

```diff
@@ -165,6 +165,100 @@ $user->store();
 // Sets the Address::$userId
 // Stores the address
 ```
+When you have multiple relations to the same class, make sure you assign
+each of them a unique role using the second parameter to avoid collisions
+between them:
+```php
+class Appointment
+{
+    use Neat\Object\Storage;
+    use Neat\Object\Relations;
+
+    public function createdBy(): Neat\Object\Relations\One
+    {
+        return $this->belongsToOne(User::class, 'creator');
+    }
+
+    public function updatedBy(): Neat\Object\Relations\One
+    {
+        return $this->belongsToOne(User::class, 'updater');
+    }
+}
+```
+
+## References
```

Comment:
Readme is inconsistent:

- Appointment model is missing it's primary key.
- The User relation remote key is not defined anywhere (`alternativeId`)

Reply:
Fixed examples, inconsistencies and inspections in readme.md and readme.php